project.yaml

```diff
@@ -5,19 +5,27 @@ expectations:
 
 actions:
   generate_cohort:
-    run: cohortextractor:latest generate_cohort --study-definition study_definition --index-date-range "2021-01-01 to 2021-06-30 by month"
+    run: >
+      cohortextractor:latest generate_cohort
```

Comment:
This does make it much nicer to read. We should encourage this styling elsewhere!

Reply:
I like this as well! with time I got used to have all configurations in one line but for beginners this styling might be easier to understand